chsdi/locale/fi/LC_MESSAGES/chsdi.po

```diff
@@ -2217,64 +2217,13 @@ msgid "ch.bafu.laerm-strassenlaerm_tag"
 msgstr "Canera da via, di"
 
 msgid "ch.bafu.landesforstinventar-baumarten"
-msgstr "Baumarten (LFI)"
+msgstr "Spezias da bostgs (IFN)"
 
 msgid "ch.bafu.landesforstinventar-totholz"
-msgstr "Totholz (LFI)"
+msgstr "Laina morta (IFN)"
 
 msgid "ch.bafu.landesforstinventar-waldanteil"
```

Comment:
I have only change one layer (energieforschung).
May be a problem with the rebase of my branch...
I do it again on Monday...